src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -281,7 +285,7 @@ class PerformanceLanding extends React.Component<Props, State> {
               start: null,
               end: null,
               utc: false,
-              period: DEFAULT_STATS_PERIOD,
+              period: isTrendsView ? DEFAULT_TRENDS_STATS_PERIOD : DEFAULT_STATS_PERIOD,
```

Comment:
Should we make performance a 14day window as well? It is an outlier in that it has a different default from other endpoints.

Reply:
iirc, I think the original concern was around performance for largest duration requests? I'm not sure if that's changed.

Reply:
Trends since it's based on time, needs larger periods of time to show more pertinent trends, the same might not be true for the rest of performance (trends is more of a weekly view then daily).

Reply:
My concern was that going between tabs currently operates on very different timescales and users could get disoriented. I could be over thinking the problem and we do need to keep the performance overview performant :)